docker/.env

```diff
@@ -30,13 +30,17 @@ IMAGE_NAME_EHSM_BASE=ehsm_base
 EHSM_VERSION_STR=main # you can change the main branch to the specific tag whit you want.
 
 # pccs url
-PCCS_URL=https://1.2.3.4:8081  # you must be adjust 1.2.3.4 to your pccs server url.
+PCCS_URL=https://10.112.240.166:8081  # you must be adjust 1.2.3.4 to your pccs server url.
 
 # Domain key server port
 DKEYSERVER_PORT=8888
 
 # domain key cache socket localtion
 DKEYCACHE_SOCKET_LOCALTION='/var/run/ehsm'
+# The role of dkeyserver has two types: root and worker
+# root: It's root node server, It will create DomainKey
+# worker: It's worker node server, It must be get a DomainKey from other dkeyserver.

```

Comment:
update it based on my reply in the chat box

Reply:
fix